config/routes.rb

```diff
@@ -80,7 +80,10 @@
   get '/account', to: 'account#index'
   get '/account/digitization_requests', to: 'account#digitization_requests', as: "digitization_requests"
   post '/account/cancel_ill_requests', to: 'account#cancel_ill_requests'
-  get '/borrow-direct', to: 'account#borrow_direct_redirect'
+
+  get '/borrow-direct', to: 'account#borrow_direct_redirect', constraints: RelaisBorrowDirectConstraint.new
```

Comment:
I like this constraints approach!

Reply:
Thanks! I hadn't seen them used before, but it seemed like the way to go.